Tests/.swiftlint.yml

```diff
@@ -4,3 +4,4 @@ disabled_rules:
   - force_unwrapping
   - nesting
   - explicit_acl
+  - missing_docs
```

Comment:
now that `TestsCommon` is in use and has public methods, we were getting missing docs errors for a few undocumented things there.